integration/pkg/tools/create_pkg.sh

```diff
@@ -24,10 +24,10 @@ chmod +x /tmp/AmazonAgentScripts/postinstall
 rm -rf artifact
 mkdir artifact
 sudo pkgbuild --root /tmp/AmazonCWAgentPackage/ --install-location "/" --scripts /tmp/AmazonAgentScripts --identifier com.amazon.cloudwatch.agent --version=$AGENT_VERSION artifact/amazon-cloudwatch-agent.pkg
-aws s3 cp ./artifact/amazon-cloudwatch-agent.pkg "s3://$1/integration-test/packaging/$2/amazon-cloudwatch-agent.pkg"
+aws s3 cp ./artifact/amazon-cloudwatch-agent.pkg "s3://$1/integration-test/packaging/$2/$3/amazon-cloudwatch-agent.pkg"
```

Comment:
Somehow the only thing when I saw using this script was uploading to S3. However, would that be the only thing we need to aware of? If not, then could we use an array to loop through or hard code it ? Since the maximum architectural we have for now is ARM64 and AMD64

Reply:
Architecture does not change very often. amd64 came out 2003 arm64 came out 2011. I don't like hard coding here since we will need 2 of this file that are almost identical.